### PR TITLE
Add aws-account-operator service

### PR DESCRIPTION
**Pull request by jharrington22**
This PR adds the deploy configuration for the aws-account-operator

### CODE REVIEW

File: aws-account-operator-services/aws-account-operator.yaml
Comment (jmelis) [inline aws-account-operator-services/aws-account-operator.yaml:4]: this path doesn't seem to exist

**Review by jmelis — CHANGES_REQUESTED** [review]

File: aws-account-operator-services/aws-account-operator.yaml
Comment (jharrington22) [reply aws-account-operator-services/aws-account-operator.yaml:4]: I have a branch in the aws-account-operator repo which contains the scripts just cleaning them up.

Comment (jharrington22) [reply aws-account-operator-services/aws-account-operator.yaml:4]: That PR is ready for review/merge https://github.com/openshift/aws-account-operator/pull/38

**Comment by jmelis** [comment]
LGTM, but would like to wait until this is merged: https://github.com/openshift/aws-account-operator/pull/38#pullrequestreview-225483256

**Review by jmelis — APPROVED** [review]

**Pull request merged**
